Backpacks.cs

```diff
@@ -651,6 +661,37 @@ public void Open(BasePlayer looter)
                     return;
                 }
 
+                if (looter.userID == OwnerId)
+                {
+                    // Check for items beyond the backpack's capacity
+                    var allowedCapacity = GetCapacity();
```

Comment:
Possibly compare this to the actual capacity and only continue if the actual capacity it larger than it should be. Or, alternatively, which would likely be a bit more performant, keep a boolean `hasOverflowingItems` variable which is set in `Initialize()` and then checked for and reset here.